app/helpers/vacancies_helper.rb

```diff
@@ -60,13 +60,7 @@ def organisation_type_value(vacancy)
   end
 
   def vacancy_or_organisation_description(vacancy)
-    vacancy.about_school.presence || vacancy.organisation.description.presence
-  end
-
-  def vacancy_about_school_value(vacancy)
-    return "" if vacancy.organisations.many?
-
-    vacancy_or_organisation_description(vacancy)
+    vacancy.organisation.description.presence
```

Comment:
I wonder if this helper is needed at all now that it is just `vacancy.organisation.description.presence`

If keeping the method, would it make sense to rename it to `vacancy_organisation_description` or `organisation_description` ?